.github/workflows/publish-flutter.yml

```diff
@@ -57,14 +57,21 @@ jobs:
         run: |
           rm -rf android
           rm -rf ios
+          rm -rf macos
           rm -rf lib
           cp -r ../build/packages/flutter/android .
           cp -r ../build/packages/flutter/ios .
+          cp -r ../build/packages/flutter/macos .
+          cp -r ../build/lib/bindings/langs/swift/Sources ios/
+          cp -r ../build/lib/bindings/langs/swift/Sources macos/          
```

Comment:
We can remove these as we are downloading the `bindings-swift` artifact, which also includes the generated binding.... but... we need to add a step after the download to copy the ios/Sources to macos/Sources